Mail/Views/Thread/MessageView.swift

```diff
@@ -25,6 +25,40 @@ import RealmSwift
 import Shimmer
 import SwiftUI
 
+// TODO: move to Core
```

Comment:
TODO: move to Core

Reply:
Moved to core, but kMail is locked on an old ios-Core, so we keep it here in the meantime.